api-go/tools/puppeteer/index.js

```diff
@@ -1,54 +1,68 @@
+const echarts = require("echarts");
+const { createCanvas } = require('canvas');
+const { fetchContributorsData, fetchMonthlyData } = require('./fetch');
+const { updateSeries } = require('./utils');
+
+const config = {
+  width: 896,
```

Comment:
is there any reason to choose this resolution? Maybe larger graph is better for people to use, especially we encode it as png but not svg